### ISSUE TITLE
WSL folders aren't watched, don't update on changes

### ISSUE THREAD

**Issue by zhawzid-ottj** [doublecmd/doublecmd#1809]
### Version

Double Commander
Version: 1.1.17 gamma
Revision: 548
Commit: f4189f213
Build date: 2024/08/19
Lazarus: 3.5.0.0
FPC: 3.2.2
Platform: x86_64-Win64-win32/win64
OS version: Windows 11 23H2 x86_64

### Description

When you have a wsl folder (like \\wsl.localhost\Ubuntu\home\) in a panel and it contents change, that panel doesn't get updated without using manual refresh.

**Comment by alexx2000** [comment]
Function not supported. It does not work in the Explorer too when content has changed by other program.

**Comment by zhawzid-ottj** [comment]
Yeah, at least you could do a re-scan when the panel receives focus.
My very old "Altap Salamander" (which I plan to replace by Double Commander) does that. 
Makes it much more convenient.

**Comment by alexx2000** [comment]
> Yeah, at least you could do a re-scan when the panel receives focus.

Such behavior has a drawback, DC will be hang temporarily when slow network directory are open in panel.

**Comment by zhawzid-ottj** [comment]
maybe as an option? 
To me, a file manager that doesn't show current content at least when I focus it, is the worse scenario.
Most of the time, a slow network folder is slow on first access or after e.g. SMTP timeout. With Servant Salamander never had issues through its behavior.

**Comment by lars18th** [comment]
Hi @alexx2000 ,

I've a suggestion to solve this limitation:
- In each panel (at tab level) you can add a togle button to enable/disable the automatic action "content refresh when focus in". And this option is disabled by default (but with a configuration to enable by default). And then with a simple "check" you can enable for this directory the "auto-focus-refresh", that is what @zhawzid-ottj is requesting (and IMHO a good idea).

What you think?
